PyPrograms/areas/readme.md

```diff
@@ -0,0 +1,12 @@
+#### THIS PROGRAM USES [GPL V3 LICENSE](../../LICENSE)
+# areas
+areas is a program you can use to find areas of different shapes.
+
+### How to run?
+Run as a python program
+
+### Requirements
+A python compiler in Windows, you can use terminal in Mac & Linux
```

Comment:
Replace compiler with interpreter because python is not a compiled language but it is a interpreted language.